app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/deployCloudFormationStackConfig.html

```diff
@@ -6,67 +6,90 @@ <h4>Basic Settings</h4>
         application="application"
         component="stage"
         accounts="ctrl.accounts"
-        show-cluster-select="ctrl.showClusterSelect">
+        show-cluster-select="ctrl.showClusterSelect"
+      >
       </account-region-cluster-selector>
     </div>
     <h4>Template Configuration</h4>
     <ng-form name="amazonCloudFormationTemplateSource">
       <stage-config-field label="Stack name">
         <label class="sm-label-right">
-          <input type="text" class="form-control input-sm" required ng-model="stage.stackName"/>
+          <input type="text" class="form-control input-sm" required ng-model="stage.stackName" />
+        </label>
+        <br />
+        <label>
+          <input type="checkbox" ng-checked="ctrl.isChangeSet()" ng-click="ctrl.toggleChangeSet()" />
+          <strong>Create CloudFormation ChangeSet</strong>
         </label>
       </stage-config-field>
+      <div ng-if="ctrl.isChangeSet()">
+        <stage-config-field label="ChangeSet name">
+          <label class="sm-label-right">
+            <input type="text" class="form-control input-sm" ng-model="stage.changeSetName" />
+          </label>
+        </stage-config-field>
+      </div>
       <stage-config-field label="Source" help-key="aws.cloudformation.source">
         <label class="sm-label-right">
-          <input type="radio" ng-model="ctrl.$scope.stage.source" value="{{ctrl.textSource}}">
+          <input type="radio" ng-model="ctrl.$scope.stage.source" value="{{ctrl.textSource}}" />
           {{ ctrl.textSource | robotToHuman }}
-          <span ng-if="stage.source === ctrl.textSource">
-          </span>
-        </label><br/>
+          <span ng-if="stage.source === ctrl.textSource"> </span> </label
+        ><br />
```

Comment:
curious if this was prettier changing the spacing here? looks kinda odd :man_shrugging:

Reply:
yes, it was intellij doing some aesthetics on my behalf without me noticing. I fixed that odd spacing 👍